nuitka/utils/Timing.py

```diff
@@ -61,7 +59,9 @@ def __enter__(self):
         self.timer.start()
 
     def __exit__(self, exception_type, exception_value, exception_tb):
+        from nuitka import Options
```

Comment:
This is typical for how you added the import in many cases. Here e.g. the stop of the timer should be the first thing, definitely not a module import. I don't like these local imports at all.

Reply:
Yeah Kay, I don't like myself either... but again, let's consider this as an intermediate PR for proper refactoring, I've already talked quite a lot about how the proper refactoring should look like so I'm not repeating myself. I'd refactor a lot of stuff all at once. But then the number of conflicts and merging would become quite chaotic :/ . So let's keep this as a set of small PRs instead

Reply:
I have not yet any idea, why making the imports local is any step in the right direction. And frankly, I am no longer convinced you do.

The runner you showed off, the me at least, seems to not need any of these changes, does it?